packages/server/src/api/controllers/application.ts

```diff
@@ -207,6 +208,7 @@ export async function fetchAppPackage(ctx: UserCtx) {
   let application = await db.get(DocumentType.APP_METADATA)
   const layouts = await getLayouts()
   let screens = await getScreens()
+  const license = await licensing.getLicense()
```

Comment:
This will actually call to account portal every time, we should use `getCachedLicense`